openfl/interface/interactive_api/experiment.py

```diff
@@ -199,8 +202,8 @@ def start(self, *, model_provider, task_keeper, data_loader,
 
         self._prepare_plan(model_provider, data_loader,
                            rounds_to_train,
-                           delta_updates=delta_updates, opt_treatment=opt_treatment,
-                           device_assignment_policy=device_assignment_policy,
+                           delta_updates=delta_updates, opt_treatment=opt_treatment.name,
+                           device_assignment_policy=device_assignment_policy.name,
```

Comment:
We may convert those later, before saving to plan or sending through gRPC
```suggestion
                           delta_updates=delta_updates, opt_treatment=opt_treatment,
                           device_assignment_policy=device_assignment_policy,
```